src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.3.td.json

```diff
@@ -0,0 +1,33 @@
+{
+  "title": "Unsuccessful search for case with none of the new global search parameters",
+  "_guid_": "S-1005.3",
+  "_extends_": "Search_Case_Base",
+  "specs": [
+    "an active profile in CCD",
+    "contains none of the fields from new global search screen",
+    "contains all the mandatory parameters",
+    "has 400 return code",
+    "contains the error message 'Must have at least one search criteria'"
+  ],
+  "request": {
+    "body": {
+      "searchCriteria": {
+      }
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "body" : {
+      "exception" : "org.springframework.web.bind.MethodArgumentNotValidException",
+      "timestamp" : "[[ANYTHING_PRESENT]]",
```

Comment:
I think you can remove this one as it comes from `Common_4XX_Response_Body`

Reply:
done 😊